### ISSUE TITLE
Add command line option to control an accessor's component type

### ISSUE THREAD

**Issue by willeastcott** [facebookincubator/FBX2glTF#129]
Allowing the user to override the default component type for an accessor would be very useful for optimizing file size.

Example:
Image you write a GLB that has 1MB of vertex data. Let's say the vertex format is:

Position: 3 * FLOAT
Normal: 3 * FLOAT
Color: 4 * FLOAT

If you could request the color attribute be written a 4 * UNSIGNED BYTE, you reduce the size of the vertex data by 3 tenths. Quite a saving!

**Comment by zellski** [comment]
Thanks @willeastcott, this is an excellent feature request. I'm sure you saw that we already implemented a related feature a while back:
```
      --long-indices arg        Whether to use 32-bit indices
                                (never|auto|always).
```
I think I would want to keep the same type of high-level control switch for this feature, e.g. `--integer-colors (never|bytes|shorts)`, and I guess the same for `--integer-texcoords` and `--integer-weights`. In practice, I imagine this would invariably go along with accessor.normalized == true.

I guess an alternative could be `--integer-attributes [colors|texcoords|weights] (never|bytes|shorts)`, where e.g. `--integer-attributes bytes` would be generously interpreted as applying to all attribute types.

Would this provide enough control? I always want to make sure that the way we add features doesn't give the user a chance to accidentally request invalid glTF.

**Comment by willeastcott** [comment]
Yes, giving that control over the vertex index type is useful!

I think starting out with an ```integer-colors``` option would be a great first step. But yes, enabling users to fully exploit [the different component types supported by the glTF 2.0 spec](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#meshes) for all vertex attributes would be very useful!

**Comment by zellski** [comment]
Alright, but outside of the variance that'd be provided with both --long-indices and --integer-attributes, what other data type modifications do you see happening? The core 2.0 spec doesn't allow us to use integers for POSITION, for example.

**Comment by willeastcott** [comment]
Correct, what you described originally does actually cover what the spec allows. 😄
